.github/workflows/mobile-ci.yml

```diff
@@ -275,22 +253,83 @@ jobs:
         working-directory: ./app
       - name: Build iOS
         run: |
-          echo "Building iOS app..."
+          echo "Building iOS app for simulator (no signing required)..."
           echo "Project: ${{ env.IOS_PROJECT_NAME }}, Scheme: ${{ env.IOS_PROJECT_SCHEME }}"
 
           WORKSPACE_PATH="ios/${{ env.IOS_PROJECT_NAME }}.xcworkspace"
 
-          # Use cached derived data and enable parallel builds for faster compilation
+          # Build for iOS Simulator to avoid code signing issues in CI
           xcodebuild -workspace "$WORKSPACE_PATH" \
             -scheme ${{ env.IOS_PROJECT_SCHEME }} \
             -configuration Release \
-            -destination "generic/platform=iOS" \
+            -sdk iphonesimulator \
+            -destination "generic/platform=iOS Simulator" \
             -derivedDataPath ios/build \
             -jobs "$(sysctl -n hw.ncpu)" \
             -parallelizeTargets \
             -quiet || { echo "❌ iOS build failed"; exit 1; }
           echo "✅ iOS build succeeded"
         working-directory: ./app
+
+  build-android:
```

Comment:
break `build-android` into it's own job so it can run in parallel with `build-ios`